### DISCUSSION TITLE
IO Error on Seaweedfs

### DISCUSSION THREAD

**Discussion by gqqnbig** [seaweedfs/seaweedfs#3124] [Q&A]
`/home/shared-la/` is my seaweed FUSE mount point. I did 7z and git operations on this mount point.

The following log shows `git commit ` encountered some IO hitch.

The same test script has been run on Glusterfs 5 times and none of these runs threw an error. I'm pretty confident the test script and my workload data have no problems.

```
7z x diffs-java.7z  255.36s user 410.81s system 8% cpu 2:18:19.91 total
Initialized empty Git repository in /home/shared-la/diffs-java/.git/
git add .
git add .  1073.50s user 692.38s system 13% cpu 3:44:12.86 total
git commit
error: invalid object 040000 4151697dca7f8f54641d62c35207f372b43c19b5 for 'repo996'
error: invalid object 040000 62940c6a02fc82a3113eb9dbb12bf51d51616b14 for 'repo997'
error: Error building trees
git commit -m 'init'  11.53s user 67.83s system 28% cpu 4:37.67 total
```

What could cause Seaweedfs to have these errors? Is it connection timeout? 

I want to tweak a parameter before starting another test.

I know you guys frequently ask for logs. I have a master log, a filer log, a mount log, three volume log. Which ones do you like? I personally don't see any errors there.

**Comment by chrislusf** [comment]
seems the console copy is messed up.
